docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,245 @@
+- Simplify strategy creation, empowering strategists and reducing the chance for errors.
+- Adding new products like ySwaps which increases swap efficiency and yJuniorTranches which allows for different risk profiles.
+
+## Smart Modules
+
+Smart Modules implement logic to be iterated until it can be made immutable. If any Smart Module fails, the vault can live without them just enough to return funds to depositors. **When Version 3 launches, the Smart Modules will replicate Version 2 vault behavior.**
+
+- **Debt Allocator:** Knows the maximum debt a vault can send to a strategy.
+- **Strategy Registry:** Handles adding and removing strategies of a vault. It can also include strategy metadata.
+- **Fee Manager:** Handles changing fees for vault operations.
+- **HealthCheck:** Guardrail vault operations so that profit & loss reporting is always under acceptable values.
+
+## Peripheral Modules
+
+Peripheral Modules are a separate layer of optional functions for the vault and strategies. They are not required but they facilitate building around yVaults and creating new strategies.
+
+- **yRouter**: Wrapper that handles deposits and withdrawals to/from the vault.
+- **Yearn Lens:** Information aggregator for off-chain apps.
+- **APY TWAP Oracle:** Reliable source of Yearn vaults’ past performance.
+- **ySwaps:** Internal swap system currently being developed. Reduces slippage thus improving net APY.
+- And any others you can come up with!
+
+# Version 3 yVault Specification
+
+## Definitions
+- **Asset:** Any ERC20-compliant token
+- **Shares:** ERC20-compliant token that tracks Asset balance in the vault for every distributor. Named yv<Asset_Symbol>
+- **Depositor:** Account that holds Shares
+- **Strategy:** Smart contract used to deposit in Protocols to generate yield
+- **Vault:** ERC4626 compliant smart contract that receives Assets from Depositors to distribute amongst the different Strategies added to the vault: manages accounting and Assets distribution. 
+- **Role:** Flags that an Account can do specific Vault actions. Can be fulfilled by a smart contract or an EOA.
+- **Accountant:** Smart contract that receives P&L reporting and returns shares and refunds to the strategy
+
+# VaultV3 Specification
+The Vault code has been designed as an unopinionated system to distribute depositors funds into different opportunities (aka Strategies) and robustly manage accounting.
+
+The depositors receive shares of the different investments that can then be redeemed or used as yield-bearing tokens.
+
+The Vault does not have a preference on any of the dimensions that should be considered when operating a vault:
+- **Decentralization**: Roles can be filled by common wallets, smart contracts like multisigs or governance modules.
+- **Liquidity**: Vault can have 0 liquidity or be fully liquid. It will depend on the parameters and strategies added.
+- **Security**: Vault managers can choose what strategies to add and how to do that process.
```

Comment:
`managers` is this just a term or an specific role/position?

Reply:
good question idk

Reply:
Ok thanks